### PR TITLE
8 view list of comments of an article

### PR DESCRIPTION
**Pull request by h-chagas**

### CODE REVIEW

File: src/components/CommentsList.jsx [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
+    const [comments, setComments] = useState([])
+    const [error, setError] = useState(null);
+
+   const { article_id } = useParams();
+
+
+    useEffect(() => {
+        showCommentsByArticle(article_id)
+        .then((data) => {
+            console.log(data.comments);
```
Comment (leekli) [inline src/components/CommentsList.jsx]: Console log to be removed.

**Review by leekli — CHANGES_REQUESTED** [review]
Good news Hugo, it works great.

BUT there's a console log, so I'm returning the pull request back to you as Haz stated earlier.

File: src/components/CommentsList.jsx (hunk 1 above)
Comment (h-chagas) [reply src/components/CommentsList.jsx]: Sorry. Totally forget about this. Already removed. Please check again (pull request resubmitted)

**Review by leekli — APPROVED** [review]
Thank you, approved.

Feel free to merge.

**Pull request merged**